apps/emqx_management/test/emqx_mgmt_api_plugins_SUITE.erl

```diff
@@ -271,7 +271,7 @@ install_plugin(FilePath) ->
             Token
         )
     of
-        {ok, {{"HTTP/1.1", 200, "OK"}, _Headers, <<>>}} -> ok;
+        {ok, {{"HTTP/1.1", 204, "No Content"}, _Headers, <<>>}} -> ok;
```

Comment:
`"No Content"` is actually content?
Doesn't `""` mean "no content" ?

Reply:
result `"No Content"` is form httpc (when code=204)